flexget/plugins/output/notify.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.event import event
+
+log = logging.getLogger('notify')
+
+HANDLED_PHASES = ['start', 'input', 'filter', 'output', 'exit']
+
+
+class Notify(object):
```

Comment:
Do we need a base class for these? Could we use the existing paradigm of registering notification plugins to a 'notification' group? Being in the notification group would mean you need to implement a 'send_notification' method, or somesuch, that gets passed the appropriate variables describing the desired notification.